docs/application/native/get-started/iot/first-app.md

```diff
@@ -0,0 +1,292 @@
+
+        ![Using the automatic build](media/build_automatic.png)
+
+        A check mark appears next to the menu option.
+
+    You can toggle the automatic build on and off by selecting **Project \> Build Automatically**.
+
+- **Manually**
+
+    The manual build means that you determine yourself when the application gets built.
+
+    To manually build the application, right-click the project in the **Project Explorer** view and select **Build Project**.
+
+    **Figure: Manually building the application**
+
+    ![Manually building the application](media/build_manual_mn.png)
+
+    Alternatively, you can also select the project in the **Project Explorer** view and do one of the following:
+
+    -   In the Tizen Studio menu, select **Project \> Build Project**.
+    -   Press the **F10** key.
+
+There are more than one build configurations. To see the currently active configuration or change it, right-click the project in the **Project Explorer** view and select **Build Configurations \> Set Active**. The default configuration is `Debug`. For more information, see [Building Applications](../../tutorials/process/building-app.md).
+
+After you have built the application, run it.
+
+<a name="run"></a>
+## Running Your Application
+
+You can run the application on Raspberry Pi.
+
+<a name="target"></a>
+### Running on a Raspberry Pi
+
+To run the application on a Raspberry Pi:
+
+1. Set the Wi-Fi of Raspberry Pi.
+
+    > [!NOTE]
+    > The Wi-Fi of the PC with the Tizen Studio installed and Raspberry Pi must be on the same local network.
+
+1. Generate an author certificate.
+
+    Before you run the application, you must [sign your application package with a certificate profile](../../../tizen-studio/common-tools/certificate-registration.md) in Tizen Studio.
+
+2. Run the application:
+    1. In the **Tools \> Device Manager**, select Raspberry Pi and change the connection status to **ON**.
+
+        ![Connected Raspberry Pi](media/connected_rpi.png)
+
+    2. In **Project Explorer** view, right-click the project and select **Run As \> Tizen Native Application**.
+
+        ![Run the application on a Raspberry Pi](media/app_run_n.png)
+
+        Alternatively, you can also select the project in the **Project Explorer** view and do one of the following:
+
+        -   Press the **Ctrl + F11** key.
+        -   Click the run icon in the toolbar.
+
+        If you have more than one connected device, select the device from the combo box in the toolbar before selecting to run the application.
+
+	3. See logs to confirm if the application launches successfully on Raspberry Pi.
+
+        ![Log of the IoT native service application](media/hellotizen_log.png)
+
+	For more information on the Log View, see the [checking Logs with Log View](https://docs.tizen.org/application/tizen-studio/common-tools/log-view/).
+
+    > [!NOTE]
+    > The application is launched using the default debug run configuration. To create and other configurations:
```

Comment:
Question. Isn't the "another" correct?

... To create another configurations:

Reply:
@mijin85cho, suggest using other:.

> The application is launched using the default debug run configuration. To create other configurations:

Reply:
@mijin85cho, yes I agree. We can use another. @ey928-lee, could we give some examples as to what other configurations?

Suggestion: 
The application is launched using the default debug run configuration. To create another configurations:

Reply:
@surbhig12  There're 4 options in Build Configuration.
![Screenshot from 2020-09-01 12-22-16](https://user-images.githubusercontent.com/69142090/91790815-98982980-ec4c-11ea-9d67-5ad69bcac214.png)

Can I user  "To create and use another configuration" instead of  "To create and other configurations"?